spec/integration/retryable_writes/shared/supports_modern_retries.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './performs_modern_retries'
+require_relative './performs_no_retries'
+
+module SupportsModernRetries
+  shared_examples 'it supports modern retries' do
+    let(:retry_writes) { true }
+
+    context 'against a standalone server' do
+      require_topology :single
+
+      before do
+        skip 'RUBY-2171: standalone topology currently uses legacy write retries ' \
```

Comment:
And if `pending` works instead of `skip` here that would be a better option I think.

Reply:
It looks like pending can only be used inside `it` blocks, so I believe skip is the appropriate option here. I've left a comment on RUBY-2171 to make sure whoever works on that ticket goes and un-skips these tests.